### DISCUSSION TITLE
With Low Latency performance mode, recording doesn't work in some devices

### DISCUSSION THREAD

**Discussion by ashfaq-opn** [google/oboe#1228] [Q&A] [answered]
I noticed that if I set performance mode to LowLatency in my input stream.

```setPerformanceMode(oboe::PerformanceMode::LowLatency)```

Recording just records a DC value, not any sound for Samsung Galaxy Note 20 (with default mic). I tested with OboeTester application, it gives the same problem with LowLatency. What is the cause of this?

Given this, how I can check which input device supports LowLatency mode and set that flag dynamically based on the check?

**Comment by philburk** [comment]
This sounds like a bug so I moved the conversation to #1229

**Comment by philburk** [comment, marked as answer]
Marking as "answered". Will track the Issue.
